content/en/docs/concepts/cluster-administration/kube-state-metrics.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Metrics for Kubernetes object state
+content_type: concept
+weight: 75
+description: >-
+   kube-state-metrics, an add-on agent to generate and expose cluster-level metrics.
+---
+
+The state of Kubernetes objects in the Kubernetes API can be exposed as metrics.
+An add-on agent called [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) can connect to the Kubernetes API server and expose a HTTP endpoint with metrics generated from the state of individual objects in the cluster.
+It exposes various information about the state of objects like labels and annotations, startup and termination times, status or the phase the object currently is in.
+For example, containers running in pods create a `kube_pod_container_info` metric.
+This includes the name of the container, the name of the pod it is part of, the namespace the pod is running in, the name of the container image, the ID of the image, the spec of the image, the ID of the running container and the ID of the pod as labels.
+
+An external component that is able and capable to scrape the endpoint of kube-state-metrics (e.g. Prometheus) can now be used to enable the following use cases.
```

Comment:
(nit) We [avoid Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) and their abbreviations.